core/src/main/scala-2.12/org/typelevel/ci/compat.scala

```diff
@@ -16,4 +16,7 @@ private[ci] object compat {
   implicit class traversableOnceSyntax[A](private val to: TraversableOnce[A]) extends AnyVal {
     def iterator: Iterator[A] = to.toIterator
   }
+
+  def unsafeWrapArray[A](arr: Array[A]): Seq[A] =
```

Comment:
Playing fast and loose here with the fact the definition of Seq also changed in 2.13.  Otherwise, we need to port ArraySeq or do an inefficient conversion.

Reply:
I think this is a good compromise.